assets/Menu/mne_open_prompt.bat

```diff
@@ -0,0 +1,10 @@
+:: This is used to initialize the bash prompt on Windows.
+@ECHO OFF
+
+call %/home/conda/feedstock_root/build_artifacts/mne-python_1692127656949/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_%\Scripts\Activate.bat
```

Comment:
This makes me think that the hack of doing `menuinst` ourselves will not work. Either `menuinst` or `constructor` must do something magical here. I'll revert for now but c5dfc2ab87da48b1e750ab3f199935f05f837e1f seems close to what we want